httpx/dispatch/connection_pool.py

```diff
@@ -106,13 +109,27 @@ def max_connections(self) -> BasePoolSemaphore:
     def num_connections(self) -> int:
         return len(self.keepalive_connections) + len(self.active_connections)
 
+    async def keepalive_timeouts(self) -> None:
```

Comment:
Wording suggestion, to match `.next_keepalive_check`?

```suggestion
    async def check_keepalive_timeouts(self) -> None:
```